website/content/v1.10/reference/configuration/siderolink/siderolinkconfig.md

```diff
@@ -23,7 +23,7 @@ apiUrl: https://siderolink.api/join?token=secret # SideroLink API URL to connect
 | Field | Type | Description | Value(s) |
 |-------|------|-------------|----------|
 |`apiUrl` |URL |SideroLink API URL to connect to. <details><summary>Show example(s)</summary>{{< highlight yaml >}}
-apiUrl: https://siderolink.api/join?token=secret
+apiUrl: https://siderolink.api/?token=secret
```

Comment:
shouldn't this be `https://siderolink.api/?jointoken=secret`?

Reply:
I guess you're right, thanks. In fact there might be more params there, but we can't get all of that here

Reply:
nit: if we want to match exactly what would be given by omni, we'd do `https://siderolink.api?jointoken=secret` without the trailing slash. I know it doesn't make any functional difference, but...

Reply:
I guess nobody needs this with Omni, they should use what Omni says (it might have more arguments). Let's assume this is for manual only